### ISSUE TITLE
Doc example errors with python on Windows

### ISSUE THREAD

**Issue by eugenesvk** [casey/just#2010]
Direct invocation works
```
$ python -c `print("foo" * 4)`
foofoofoofoo
```

But the example from the ReadMe fails
```just
# use python3 to execute recipe lines and backticks
set shell := ['python', "-c"]
# use print to capture result of evaluation
foos := `print("foo" * 4)`
foo:
  print("{{foos}}")

```

Has a strange error I don't understand the cause of, why did the closing `"` disappear, and what's up with `")int("`?
```sh
just foo
")int("foofoofoofoo
  File "<string>", line 1
    print("foofoofoofoo
          ^
SyntaxError: unterminated string literal (detected at line 1)
error: Recipe `foo` failed on line 6 with exit code 1
```

Python 3.12, Windows 10, tried in cmd, pwsh,

**Comment by casey** [comment]
Very strange! This seems to work for me on my Mac. Guessing that the trailing newline isn't being stripped from `foo`s, so the code winds up being:

```
print("foofoofoofoo
")
```

Which is a syntax error in Python.

**Comment by casey** [comment]
I'm able to reproduce this [on CI](https://github.com/casey/just/actions/runs/9087077066/job/24974099471#step:5:1172). I wish I could go back in time and convince everyone to just use `\n` for line breaks T_T

**Comment by casey** [comment]
Fixed in #2035! Turns out this was a pretty trivial bug on my part.

The code that stripped newlines from output was:

```rust
if utf8.ends_with('\n') {
  &utf8[0..utf8.len() - 1]
} else if utf8.ends_with("\r\n") {
  &utf8[0..utf8.len() - 2]
} else {
  utf8
}
```

Which will never strip `\r\n`, since `\n` is tested first.

**Comment by casey** [comment]
Thanks for reporting this! I'm kind of amazed that it took so long for this bug to be reported.
